components/ControlPanel/ControlPanel.tsx

```diff
@@ -123,6 +126,31 @@ export const ControlPanel: React.FC<ControlPanelPropsI> = ({
     }
   };
 
+  const submitResignGameHandler: React.FormEventHandler<HTMLFormElement> = async (event) => {
+    event.preventDefault();
+
+    // @ts-ignore TODO: to solve once it make sense
+    const gameId = event.target.children.gameId.value;
+    try {
+      if (!currentPlayerAddress)
+        throw new Error(`No currentPlayerAddress: ${currentPlayerAddress}`);
+      if (!gameId || gameId.length === 0) throw new Error(`Empty game id`);
```

Comment:
Moar data in logs pls: does it happen when we try to resign? How do we understand it from logs?